src/Lean/Meta/Tactic/Simp/SimpTheorems.lean

```diff
@@ -38,16 +38,11 @@ structure SimpTheorem where
     `name?` is mainly relevant for producing trace messages.
```

Comment:
Docstring still contains `name?`. Note that it says that we use the name for producing trace messages, and it seems you are using free variable unique identifiers as names.